java/spark/src/main/java/com/logicalclocks/hsfs/spark/util/StorageConnectorUtils.java

```diff
@@ -278,4 +288,24 @@ public Dataset<Row> readStream(StorageConnector.KafkaConnector connector, String
     return SparkEngine.getInstance().readStream(connector, connector.sparkFormat,
         messageFormat.toLowerCase(), schema, options, includeMetadata);
   }
+
+  public String validateAdlsPath(StorageConnector.AdlsConnector connector, String path) throws FeatureStoreException {
```

Comment:
why in Python this same method is part of the `AdlsConnector`, but in Java its in `StorageConnectorUtils`?

Reply:
@bubriks We had it same for java in the past. But there were some refactoring to  the return type of read method which caused to return Object instead so it was moved in another class. Davit knows better he worked on it.